webpack/assets/javascripts/react_app/components/ConfigReports/DiffModal/DiffModalActions.js

```diff
@@ -0,0 +1,15 @@
+import { DIFF_MODAL_TOGGLE, DIFF_MODAL_CREATE } from './DiffModalConstants';
+
+export const toggleModal = () => ({
+  type: DIFF_MODAL_TOGGLE,
+});
+
+export const createDiff = (patch, log) => (dispatch) => {
+  dispatch({
+    type: DIFF_MODAL_CREATE,
+    payload: {
+      patch,
```

Comment:
perhaps call it diff across ? while they both mean the same thing we do call the component diff and not patch.